src/pool/srv_target.c

```diff
@@ -920,20 +937,22 @@ ds_pool_start(uuid_t uuid)
 
 	pool = pool_obj(llink);
 
-	rc = dss_ult_create(pool_fetch_hdls_ult, pool, DSS_XS_SYS,
-			    0, 0, NULL);
-	if (rc != 0) {
-		D_ERROR(DF_UUID": failed to create fetch ult: %d\n",
-			DP_UUID(uuid), rc);
-		D_GOTO(failure_pool, rc);
-	}
+	if (!engine_in_check()) {
```

Comment:
[Question] Would it make sense to comment on why we don't want to do these in the "check mode"? To be honest, I don't know the reason.

Reply:
Because the CR logic will evict all former connections. I will add comment in subsequent patch.